GimbalAdapter.swift

```diff
@@ -47,49 +50,51 @@ open class GimbalAdapter {
      * Restores the adapter. Should be called in didFinishLaunchingWithOptions.
      */
     open func restore() {
-        if (GMBLPlaceManager.isMonitoring()) {
-            start(nil);
-        }
+        isStarted = Gimbal.isStarted()
+        placeManager.delegate = gimbalDelegate
```

Comment:
Why do we remove the adapter when its stopped? Maybe its legacy code that I brought over?